.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "src"]
 	path = src
 	url = https://github.com/fabiangreffrath/crispy-doom
+	branch = master
```

Comment:
Should we maybe point at a specific commit (latest in `master` is fine) for reproducibility?